### PR TITLE
Remove processing_status from predicate list in drop fetch query

### PR DESCRIPTION
**Pull request by ekala**
- The drop api doesn't return the list of newly added drops. Only the
  status code is returned; 200 on success 500 or 4xx otherwise
- Purge the cache whenever there's a change in the user's rivers or
  bucket listing; user accepts collaboration invite, creates/deletes a
  river/bucket
- Limit the general drop search to the last 14 days

### CODE REVIEW

File: application/classes/model/droplet.php [hunk 1]
```diff
@@ -232,9 +232,7 @@ public static function create_from_array($droplets)
 		Model_Media::get_media($droplets);
 		
 		// Populate the drop's metadata tables
-		self::add_metadata($droplets);
-		
-		return array($droplets, $new_droplets);
```
Comment (69mb) [inline application/classes/model/droplet.php:237]: The bookmarklet plugin needs this return value

**Comment by 69mb** [comment]
Just restore the return value to Model_Droplet::create_from_array and this is good to go

**Comment by ekala** [comment]
Restored.

**Comment by 69mb** [comment]
+1

**Pull request merged**